packages/mui-material/src/SpeedDialAction/SpeedDialAction.test.js

```diff
@@ -21,6 +25,13 @@ describe('<SpeedDialAction />', () => {
       testRootOverrides: { slotName: 'fab' },
       testVariantProps: { tooltipPlacement: 'right' },
       skip: ['componentProp', 'componentsProp'],
+      slots: {
+        fab: {
+          expectedClassName: classes.fab,
+          testWithElement: null,
+          testWithComponent: CustomButton,
+        },
```

Comment:
Are other slots missing?

Reply:
added `tooltip` slot, but for other slots to render `tooltipOpen` should be true, since `SpeedDialAction` [describeConference](https://github.com/mui/material-ui/pull/45065/files#diff-f29e1664b4ccb64dad9d20650cf010182a6e9e7fd3d95e98bd181bc123bc37d6R21) doesn't support both, I've written tests for other slots seperatly here

https://github.com/mui/material-ui/pull/45065/files#diff-f29e1664b4ccb64dad9d20650cf010182a6e9e7fd3d95e98bd181bc123bc37d6R149